independent-publisher-connectors/Wordwide Holidays/apiDefinition.swagger.json

```diff
@@ -0,0 +1,475 @@
+            "x-ms-summary": "Year",
+            "x-ms-url-encoding": "single",
+            "description": "Year of the public holiday"
+          },
+          {
+            "in": "path",
+            "name": "countryCode",
+            "required": true,
+            "type": "string",
+            "x-ms-summary": "Country Code",
+            "x-ms-url-encoding": "single",
+            "description": "A valid country code such as GB and UA"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "type": "array",
+              "items": {
+                "$ref": "#/definitions/LongWeekendV3Dto"
+              }
+            }
+          }
+        },
+        "description": "Get long weekends for a given country"
+      }
+    },
+    "/api/v3/PublicHolidays/{year}/{countryCode}": {
+      "get": {
+        "tags": [
+          "PublicHoliday"
+        ],
+        "summary": "Get public holidays",
+        "operationId": "GetPublicHolidays",
+        "produces": [
+          "text/plain",
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "path",
+            "name": "year",
+            "format": "int32",
+            "required": true,
+            "type": "integer",
+            "x-ms-summary": "Year",
+            "x-ms-url-encoding": "single",
+            "description": "Year of the public holiday"
+          },
+          {
+            "in": "path",
+            "name": "countryCode",
+            "required": true,
+            "type": "string",
+            "x-ms-summary": "Country Code",
+            "x-ms-url-encoding": "single",
+            "description": "A valid country code such as GB and UA"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Public holidays",
+            "schema": {
+              "type": "array",
+              "items": {
+                "$ref": "#/definitions/PublicHolidayV3Dto"
+              }
+            }
+          },
+          "400": {
+            "description": "Validation failure"
+          },
+          "404": {
+            "description": "CountryCode is unknown"
+          }
+        },
+        "description": "Get public holidays"
+      }
+    },
+    "/api/v3/IsTodayPublicHoliday/{countryCode}": {
+      "get": {
+        "tags": [
+          "PublicHoliday"
+        ],
+        "summary": "Is today a public holiday",
+        "description": "The calculation is made on the basis of UTC time to adjust the time please use the offset.<br />\r\nThis is a special endpoint for `curl`<br /><br />\r\n200 = Today is a public holiday<br />\r\n204 = Today is not a public holiday<br /><br />\r\n`STATUSCODE=$(curl --silent --output /dev/stderr --write-out \"%{http_code}\" https://date.nager.at/Api/v2/IsTodayPublicHoliday/AT)`<br /><br />\r\n`if [ $STATUSCODE -ne 200 ]; then # error handling; fi`",
```

Comment:
@reshmee011 all descriptions should be in a good sentence format and I would suggest you to keep it short to fit in the text box. Please see our docs [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/connector-string-guidance#operations).